### ISSUE TITLE
about yolo26pose

### ISSUE THREAD

**Issue by NeKoooo233** [ultralytics/ultralytics#23347]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hello, currently we define a box plus 6 key points (T) on the dataset, the test results of training show that the prediction order is correct, but the position is not very accurate (as shown in the figure), currently training according to the default parameters, imgsz: 1600, training set 224, test set 56, divided into 3 categories, shoe4 (4 key points), halfshoe6, shoe6.
What are the possible reasons for the inaccurate prediction of this key point location?

### Additional

<img width="1955" height="1079" alt="Image" src="https://github.com/user-attachments/assets/9f7f9168-d71d-463a-a96e-7d0a8461c117" />

![Image](https://github.com/user-attachments/assets/c7c17c13-bc9a-4c49-bed4-8528f2a9afbc)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @NeKoooo233, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

Thanks for sharing the screenshots and details about your YOLO26 pose setup (box + 6 keypoints). To help the team pinpoint *why* keypoint locations look shifted/inaccurate, could you please add a bit more reproducible context (config + logs) below? 🔍🧩  
Also note: this is an automated response, and an Ultralytics engineer will also assist you soon.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## What to include (so we can reproduce) 🧪
-Your exact training command (Python or CLI) and all args you changed (even if “default”)
-Your `results.png` and the full training log output (or a copy/paste of the key sections)
-A few *raw* training images + their labels showing the keypoints (and one or two of the problematic predictions)
-Your dataset YAML (or at least: `kpt_shape`, `flip_idx`, class names, and splits)
-Exact versions: `pip show ultralytics`, PyTorch version, CUDA info, OS

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
The keypoints need to be uniquely placed in the same location every time and `flip_idx` in `data.yaml` needs to be correct.

https://community.ultralytics.com/t/detecting-court-keypoints-from-a-pickleball-video/1703/7
